includes/class-wc-payments-checkout.php

```diff
@@ -332,7 +332,7 @@ public function get_enabled_payment_method_config() {
 					'number' => '<button type="button" class="js-woopayments-copy-test-number" aria-label="' . esc_attr( __( 'Click to copy the test number to clipboard', 'woocommerce-payments' ) ) . '" title="' . esc_attr( __( 'Copy to clipboard', 'woocommerce-payments' ) ) . '"><i></i><span>',
 				]
 			);
-			$settings[ $payment_method_id ]['forceNetworkSavedCards'] = $gateway_for_payment_method->should_use_stripe_platform_on_checkout_page();
+			$settings[ $payment_method_id ]['forceNetworkSavedCards'] = WC_Payments::is_network_saved_cards_enabled() || $gateway_for_payment_method->should_use_stripe_platform_on_checkout_page();
```

Comment:
As per another comment, I think it would be more safe to move the check here as shown below (or a similar syntax which achieves the same result):

```suggestion
			$settings[ $payment_method_id ]['forceNetworkSavedCards'] = 
				'card' === $payment_method_id && 
				(WC_Payments::is_network_saved_cards_enabled() || 
				$gateway_for_payment_method->should_use_stripe_platform_on_checkout_page());
```

Reply:
Makes sense to me. Addressed in [2c840ae](https://github.com/Automattic/woocommerce-payments/pull/9460/commits/2c840ae61a01ab2997b3e8cb289f1eb4a11839b1)